Automattic/wp-calypso
# Domains: Update subdomain connection instructions

### Changes proposed in this Pull Request

Up until now we were showing the same instructions to connect domains and subdomains, which could be confusing for customers who wanted to connect the latter, as they could mistakenly change their root domain's records or name servers. Cases like this were mentioned by HEs such as in p2MSmN-96M-p2.

This PR updates the instructions to connect subdomains to WPCOM, making them clearer and different from the instructions to connect root domains.

The following screenshots show the changes in the steps of both the suggested and advanced connection setup flows:

- Suggested setup

| Before | After |
| --- | --- |
| ![Markup on 2022-02-15 at 20:19:00](https://user-images.githubusercontent.com/5324818/154166540-144d7509-66cf-44da-b342-e360a84ad6c2.png) | ![Markup on 2022-02-25 at 16:36:07](https://user-images.githubusercontent.com/5324818/155783865-795dc170-a3fa-4594-8248-b89990cc91c1.png) |
| <img width="1067" alt="154166711-629a09a2-fa8b-401c-ae10-d5cecd9cd7d9" src="https://user-images.githubusercontent.com/5324818/155628586-2c9ba6ba-70b9-4505-92d5-63e223849b89.png"> | <img width="717" alt="Screen Shot 2022-02-25 at 16 37 44" src="https://user-images.githubusercontent.com/5324818/155784026-e8090c3e-54ed-4240-8265-9c6d7c6d2c9f.png"> |

- Advanced setup

| Before | After |
| --- | --- |
| ![Markup on 2022-02-15 at 20:19:31](https://user-images.githubusercontent.com/5324818/154166880-707aaa35-43dd-45c2-a264-8e838f16b38f.png) | ![Markup on 2022-02-25 at 16:41:33](https://user-images.githubusercontent.com/5324818/155784476-721859eb-4404-4ede-9dcc-f01d7d8dde1f.png) |
| <img width="1066" alt="Screen Shot 2022-02-15 at 20 19 48" src="https://user-images.githubusercontent.com/5324818/154166934-6f45d81b-e359-46c8-88f7-9a3a11d68d55.png"> | <img width="761" alt="Screen Shot 2022-02-24 at 21 23 39" src="https://user-images.githubusercontent.com/5324818/155630084-4c3606f2-67bc-49a3-a7db-be8867a46dbd.png"> |

### Testing instructions

- Build this branch locally or open the live Calypso link
- Connect a new subdomain to a site to access the connection setup, or access an already connected subdomain's setup via `/domains/mapping/<site_slug>/setup/<subdomain>`
- Ensure the instructions and recommended DNS record configurations were updated like in the screenshots above, both in the suggested and advanced flows
- Please also test connecting a root domain and ensure the instructions are the same as they were before
